Inventory_Management_System/InventoryManagement/Inventory.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Inventory_Management_System.ProductManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Inventory_Management_System.InventoryManagement
+{
+    public class Inventory
+    {
+        private List<Product> products = new List<Product>();
+        // Add a product to the inventory
+        public void AddProduct(string name, decimal price, Currency currency, int quantity)
```

Comment:
It is not a good pracrtise to pass number of parameters, try passing the whole object